### ISSUE TITLE
Add Breadcrumb pattern

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-web-components#901]
https://opensource.adobe.com/spectrum-css/breadcrumb.html

![image](https://user-images.githubusercontent.com/1156657/94156822-845fda80-fe4e-11ea-96bb-19a3c837f3ac.png)

Teams are looking for API details (even pre-implementation). This issue is a good place to start hashing that out.

Starting from a [reasonable complex selection](https://opensource.adobe.com/spectrum-css/breadcrumb.html#multilinenestedrootvisible) from the examples, I'd initially see this looking a bit like:

```
<sp-breadcrumbs multiline>
  <sp-breadcrumbs-item
    href="#" (likely powered by `LikeAnchor`)
  > <!-- room for sp-breadcrumb vs sp-breadcrumbs-item discussion here -->
    Nav Root
  </sp-breadcrumbs-item>
  <sp-breadcrumbs-nested
    (this is essentially a proxy for `sp-action-menu` as seen here: https://spectrum.adobe.com/page/breadcrumbs/#Behaviors. Having it's own element means we don't have to require the import of that for every user of `sp-breadcrumbs`)
  >
    <span slot="label">I don't see accessible text in the current Spectrum CSS demo will investigate.</span>
    <sp-menu>
        <sp-menu-item>
            Deselect
        </sp-menu-item>
        <sp-menu-item>
            Select inverse
        </sp-menu-item>
        <sp-menu-item>
            Feather...
        </sp-menu-item>
        <sp-menu-item>
            Select and mask...
        </sp-menu-item>
        <sp-menu-divider></sp-menu-divider>
        <sp-menu-item>
            Save selection
        </sp-menu-item>
        <sp-menu-item disabled>
            Make work path
        </sp-menu-item>
    </sp-menu>
  </sp-breadcrumbs-nested>
  <sp-breadcrumbs-item
    href="#"
  >
    Trendy
  </sp-breadcrumbs-item>
  <sp-breadcrumbs-item
    href="#"
    current (room for conversation as to whether this is applied manually or resolved dynamically)
  >
    January 2019 Assets
  </sp-breadcrumbs-item>
</sp-breadcrumbs>
```

cc: @kerrishotts

**Comment by AndreasGalster** [comment]
Hi @Westbrook I'd be interested to work on this component. I think it's relatively simple and it would be a good exercise for me to

- learn typescript
- understand how you structure your components
- get to know spectrum
  - I think that's the most daunting part of how you build your components, I've read some of your other issues and have no idea what you folks are talking about in terms of spectrum terminology 😄 

Thoughts?

**Comment by Westbrook** [comment]
@AndreasGalster, I'd love to see where you were able to get taking on this pattern. I won't lie and tell you it's a simple pattern, there are a lot of features in this pattern. However it's layered, so if you start with the [Standard](https://opensource.adobe.com/spectrum-css/breadcrumb.html#standard) version to get a handle of things and work up from there, I think you'd be able to find a good balance of challenge and results.

To that end, let's get started with a couple of things:
1. I've created a [`breadcrumb`](https://github.com/adobe/spectrum-web-components/tree/breadcrumb) that can work as your target branch for a PR, and a base branch for the component/project as a whole in case there are things that maybe you'd like support in preparing along the way. This will also give us a holding branch to build any project specific golden image caches before going into `main`.
2. I'd like you to have some golden image caches local to your repo before we started something like this, but as we've discussed, I've not really prepared a process around this for external contributors, yet. In theory, if you create a PR to the above branch today, even if it's just a "hello world" PR you'll be creating a golden image cache that is accessible from your repo and you can update your local and then you can use these instructions to update the cache your local branch points to: https://github.com/adobe/spectrum-web-components#keeping-ci-assets-up-to-date. However, I'm testing this process for the first time with #903. 
3. Beyond that, I would like to have at least a PR that you can review for #590 as it will help outline some of the steps you'll be taking along this process. I'll work to get something together on this by Monday so you can check it out as you start leaning into this.

I have a little bit of an understanding as to where you are in your process, so if you have a little patience with us as we work to better support external contributors, I think we could find some nice success with this project. Along the way, ask questions here or on Slack, and we'll find the best way to keep you moving forward!

Sound good?

**Comment by AndreasGalster** [comment]
Yes sounds perfect :) Thanks for all the info! Looking forward to the info from #590

**Comment by Westbrook** [comment]
This is no longer actively on our roadmap, feel free to open a new issue if you feel it is important to impement.
